packages/client-websocket/src/core-pikku-websocket.ts

```
@@ -61,14 +79,22 @@ export class CorePikkuWebsocket {
 
   private handleMessage(event: MessageEvent) {
-    let parsed = false
-
     if (!event.data) {
       // This occurs because aws seems to be returning
       // a message with no data
       return
     }
 
+    if (event.data instanceof ArrayBuffer) {
+      this.binarySubscriptions.forEach((cb) => cb(event.data as ArrayBuffer))
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)